joomla/joomla-cms
# [4.0] Fix multiple sampledata

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/20662

### Summary of Changes
Adds a check to the sampledata JavaScript to ignore empty responses in the foreach loop.

I'm not sure what the current process is to generate the minimised JS files. Maybe @C-Lodder or @dgrammatiko can help here.

### Testing Instructions
* Install the testing sampledata plugin (or any other one) from https://github.com/joomla-extensions/testing-sample-data/releases/tag/v3.8.0. That one is expected to be broken in 4.0 but for this test it will do.
* Try installing the "Blog Sample Data" from the Control Panel.

### Expected result
Works

### Actual result
A JS Error prevents more than one step executed. See original issue for screenshots.

### Documentation Changes Required
None
